db/memtable.cc

```diff
@@ -170,6 +172,11 @@ size_t MemTable::ApproximateMemoryUsage() {
 }
 
 bool MemTable::ShouldFlushNow() {
```

Comment:
It is possible to log the reason for flush? Maybe consider adding the reason to `FlushReason `.

Reply:
Okay, let me see if i can identify where to generate the flush reason.